A/aws_c_s3/build_tarballs.jl

```diff
@@ -36,9 +36,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("aws_checksums_jll"; compat="0.2.8"),
-    Dependency("aws_c_http_jll"; compat="0.10.2"),
-    Dependency("aws_c_auth_jll"; compat="0.9.1"),
+    Dependency("aws_checksums_jll"; compat="0.2.7"),
+    Dependency("aws_c_http_jll"; compat="0.10.1"),
+    Dependency("aws_c_auth_jll"; compat="0.9.0"),
```

Comment:
@quinnj updating the compat was a good idea, but I think there's something wrong in the logic of the update: this is _downgrading_ them.  This is happening in a bunch of other PRs: https://github.com/JuliaPackaging/Yggdrasil/issues?q=sort%3Aupdated-desc+is%3Apr+is%3Aopen+author%3Aquinnj2